src/screens/PutawayCandidates/index.tsx

```diff
@@ -21,13 +21,25 @@ class PutawayCandidates extends Component<Props> {
 
     this.state = {
       refreshing: false,
+      updatedlist: [],
     };
   }
 
   UNSAFE_componentWillMount() {
     this.getScreenData();
   }
 
+  componentDidUpdate() {
+    const {candidates} = this.props;
+    let updatedlist: any = [];
+    if (candidates.length) {
```

Comment:
You don't need this check
Please assign it directly to the `updatedList` and use `const` instead `let` since it won't change.
Change:
`let updatedlist: any = [];`
Into:
`const updatedList = candidates.filter(candidate => candidate.putawayStatus === 'READY');`
and remove candidates length if statement